### DISCUSSION TITLE
Geofencing

### DISCUSSION THREAD

**Discussion by RJGhugo** [streetcomplete/StreetComplete#5270] [Feedback]
Using StreetComplete it is not possible to add information outside of a certain range.

a) Adding Basis
I am on the road and see something missing (e.g. restaurant). I add it via StreetComplet.

b) At Home
At home I would like to add more information about the place (e.g. web-site, opening times)

How do I do this?

**Comment by HolgerJeromin** [comment]
The title is a bit confusing here...

> a) Adding Basis
> I am on the road and see something missing (e.g. restaurant). I add it via StreetComplet.

You can add such things in the shop overlay. This is now available via the layer icon on the main screen.

> b) At Home
> At home I would like to add more information about the place (e.g. web-site, opening times)

Please use https://every-door.app for that.

**Reply by matkoniecz** [reply]
or iD or JOSM or Vespucci...

**Comment by RJGhugo** [comment]
Thanks for the hint. I am using now "every door".
